Corona-Warn-App/src/main/res/values-ar/strings.xml

```diff
@@ -0,0 +1,1024 @@
+
+    <!-- ####################################
+               App Information
+    ###################################### -->
+
+    <!-- XHED: Page title for menu information page -->
+    <string name="information_title">"معلومات التطبيق"</string>
+    <!-- XTXT: Version information text -->
+    <string name="information_version">"الإصدار:%1s"</string>
+    <!-- XHED: Page title for about information page, also menu item / button text -->
+    <string name="information_about_title">"نبذة عن التطبيق"</string>
+    <!-- XHED: Subtitle for about information page -->
+    <string name="information_about_headline">"لنكافح فيروس كورونا معًا"</string>
+    <!-- YTXT: Body text for about information page -->
+    <string name="information_about_body_emphasized">"معهد روبرت كوخ (RKI) هو هيئة مركزية ألمانية معنية بالصحة العامة في الولايات الاتحادية الألمانية. ينشر المعهد تطبيق Corona-Warn-App لصالح الحكومة الاتحادية. ويمثل التطبيق الامتداد الرقمي لإجراءات الصحة العامة المنفَّذة بالفعل: التباعد الاجتماعي والنظافة الشخصية واستخدام أقنعة الوجه."</string>
+    <!-- YTXT: Body text for about information page -->
+    <string name="information_about_body">"يمكن لأي شخص يستخدم التطبيق تتبع سلسلة العدوى وكسرها. يحفظ التطبيق التجمُّعات مع الأشخاص الآخرين محليًا على جهازك. ويتم إشعارك عند تجمُّعك مع أشخاص تم تشخيصهم مؤخرًا على أنهم مصابون بمرض كوفيد-19، علمًا أن هويتك وخصوصيتك محميتان دائمًا."</string>
+    <!-- XACT: describes illustration -->
+    <string name="information_about_illustration_description">"مجموعة أشخاص يستخدمون أجهزتهم في المدينة جميعًا."</string>
+    <!-- XHED: Page title for privacy information page, also menu item / button text -->
+    <string name="information_privacy_title">"خصوصية البيانات"</string>
+    <!-- XACT: describes illustration -->
+    <string name="information_privacy_illustration_description">"إحدى السيدات تستخدم تطبيق Corona-Warn-App على جهازها. القفل الموجود على الشاشة هو رمز للبيانات المشفرة."</string>
+    <!-- XTXT: Path to the full blown privacy html, to translate it exchange "_de" to "_en" and provide the corresponding html file -->
+    <string name="information_privacy_html_path">"privacy_en.html"</string>
+    <!-- XHED: Page title for terms of use information page, also menu item / button text -->
+    <string name="information_terms_title">"شروط الاستخدام"</string>
+    <!-- XHED: Page headline for terms of use information page -->
+    <string name="information_terms_headline">"المحتوى"</string>
+    <!-- XACT: describes illustration -->
+    <string name="information_terms_illustration_description">"يحمل شخص ما جهازًا به الكثير من النصوص على الشاشة. بجوار النص توجد علامة صح تمثل رمزًا لقبول شروط الاستخدام."</string>
+    <!-- XTXT: Path to the full blown terms html, to translate it exchange "_de" to "_en" and provide the corresponding html file -->
+    <string name="information_terms_html_path">"terms_en.html"</string>
```

Comment:
Just a friendly reminder to point out a potential inconsistency:
Linking to `terms_en.html` makes sense, when their is no full translation (yet),
but note that in the `ru` translation in this PR, this links is updated to `terms_ru.html`.